context/app/static/js/helpers/functions.js

```diff
@@ -0,0 +1,9 @@
+export function isEmptyArrayOrObject(val) {
+  let ret = false;
+  if (val.constructor.name === 'Object') {
+    ret = Object.keys(val).length === 0;
+  } else if (val.constructor.name === 'Array') {
+    ret = val.length === 0;
+  }
+  return ret;
```

Comment:
Avoid `let` and just return as soon as possible?
```suggestion
  if (val.constructor.name === 'Object') {
    return Object.keys(val).length === 0;
  }
  if (val.constructor.name === 'Array') {
    return val.length === 0;
  }
  return false;
```